frameworks/kafka/universe/config.json

```diff
@@ -8,43 +8,158 @@
           "name" : {
             "description":"Apache Kafka",
             "type":"string",
-            "default":"kafka"
+            "default":"apache-kafka"
           },
           "mesos_api_version" : {
             "description":"Configures the Mesos API version to use. Possible values: V0 (non-HTTP), V1 (HTTP)",
             "type":"string",
-            "default":"V0"
+            "default":"V1"
```

Comment:
Please don't default to V1.  We are just starting to test the V1 API.  We don't want it used by Kafka yet.